lib/index.js

```diff
@@ -323,7 +312,7 @@ exports.getAll = brCallbackify(async (
  *
  * @returns {Promise} Resolves once the operation completes.
  */
-exports.update = brCallbackify(async ({actor, id, patch, sequence}) => {
+exports.update = async ({actor, id, patch, sequence}) => {
```

Comment:
This is a common oversight throughout our code base.  Without default the parameter like this, the validation on L316-318 does not work as intended because a destructuring error occurs if a dev calls `update()` without any parameters.  Please be aware of this and update any time you notice it.

```suggestion
exports.update = async ({actor, id, patch, sequence} = {}) => {
```